packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -236,7 +236,7 @@
         <value>Could not connect.</value>
         <language>en_US</language>
         <protected>false</protected>
-        <shortDescription>Could not connect.</shortDescription>
+        <shortDescription>Couldn't connect.</shortDescription>
```

Comment:
Why did you change this description? `Could not connect.` seemed correct to me.

Reply:
Nothing major, it's simply because we now accept contractions in our docs.